### PR TITLE
feat(electron): Add details of JavaScript framework specific init

### PR DESCRIPTION
**Pull request by timfish**
Adds details of how to configure the Electron SDK combined with a JavaScript framework specific SDK.

### CODE REVIEW

File: src/platforms/javascript/guides/electron/index.mdx [hunk 1]
```diff
@@ -70,6 +70,18 @@ In the renderer processes:
 import * as Sentry from "@sentry/electron/renderer";
 ```
 
+## Framework Specific SDKs
```
Comment (imatwawana) [inline src/platforms/javascript/guides/electron/index.mdx:73]:
```suggestion
## Framework-Specific SDKs
```

File: src/platforms/javascript/guides/electron/index.mdx [hunk 2]
```diff
@@ -70,6 +70,18 @@ In the renderer processes:
...
+
+If you are using a framework specific Sentry SDK in the Electron renderers, you
+can pass that `init` function as the second parameter and the two SDKs functionalities will be combined:
```
Comment (imatwawana) [inline src/platforms/javascript/guides/electron/index.mdx:76]:
```suggestion
If you're using a framework-specific Sentry SDK in the Electron renderers, you
can pass that `init` function as the second parameter and the two SDKs functionalities will be combined:
```

**Review by imatwawana — COMMENTED** [review]
Just a couple small grammar things

**Comment by timfish** [comment]
This can be merged when [this](https://github.com/getsentry/publish/issues/1464) has been released!

**Comment by timfish** [comment]
This has now made it into release 4.1.0 so can be merged whenever it's ready!

**Pull request merged**